src/llm-models/codestral-completer.ts

```diff
@@ -15,99 +19,73 @@ import { BaseCompleter, IBaseCompleter } from './base-completer';
 const INTERVAL = 1000;
 
 /**
- * Timeout to avoid endless requests
+ * The default prompt for the completion request
+ * TODO: move somewhere else so it can be used by other completers
+ * See: https://github.com/jupyterlite/ai/issues/25
  */
-const REQUEST_TIMEOUT = 3000;
+const DEFAULT_PROMPT = `
```

Comment:
Looks like we can now update this PR to use the default prompts from https://github.com/jupyterlite/ai/pull/28?